frontend/src/App.jsx

```diff
@@ -262,7 +262,6 @@ const App = () => {
                     path="create-assessment"
                     element={<CreateAssessment />}
                   />
-                  <Route path="admin-csv-upload" element={<AdminCSVUpload />} />
```

Comment:
You removed this route, but this route is needed for the upload admincsvupload. can you add it back?